openlibrary/templates/widget.html

```diff
@@ -27,24 +27,24 @@ <h2 class="author">
       $if book_status == "open" :
         $ public_url = canonical_url("/books/" + item["availability"]["openlibrary_edition"] + "/x/borrow")
         <div class="cta open" type="submit">
-          <a data-ol-link-track="Embed|Read"  target="_blank" href="$public_url" title="$_('Read \"%(title)s\"', title=book_title)">$_('Read')</a>
+          <a data-ol-link-track="Embed|Read"  target="_blank" href="$public_url" title="$_('Read \"%(title)s\"', title=book_title)">$_("Read")</a>
 	</div>
       $elif book_status == "borrow_available" :
         $ borrow_url = canonical_url("/borrow/ia/" + item["availability"]["identifier"])
         <div class="cta available" type="submit">
-          <a data-ol-link-track="Embed|Borrow" target="_blank" href="$borrow_url" title="$_('Borrow \"%(title)s\"', title=book_title)">$_('Borrow')</a>
+          <a data-ol-link-track="Embed|Borrow" target="_blank" href="$borrow_url" title="$_('Borrow \"%(title)s\"', title=book_title)">$_("Borrow")</a>
 	</div>
       $elif book_status == "borrow_unavailable":
         $ waitlist_url = canonical_url("/borrow/ia/" + item["availability"]["identifier"] + "?action=join-waitinglist")
         <form class="cta waitlist" method="POST" action="$waitlist_url" target="_blank">
           <input type="hidden" name="action" value="join-waitinglist">
-          <button data-ol-link-track="Embed|JoinWaitlist" type="submit" title="$_('Join waitlist for &quot;%(title)s&quot;',title=book_title)">$_('Join Waitlist')</button>
+          <button data-ol-link-track="Embed|JoinWaitlist" type="submit" title="$_('Join waitlist for \"%(title)s\"', title=book_title)">$_("Join Waitlist")</button>
         </form>
       $else:
         <div class="cta open">
-          <a data-ol-link-track="Embed|Learn" target="_blank" href="$(canonical_url(item['key']))" title="$_('Learn more about &quot;%(title)s&quot; at OpenLibrary', title=book_title)">$_('Learn More')</a>
+          <a data-ol-link-track="Embed|Learn" target="_blank" href="$(canonical_url(item['key']))" title="$_('Learn more about \"%(title)s\" at OpenLibrary', title=book_title)">$:_('Learn More</a>
 	</div>
-      <p class="service">$_('on ')<a target="_blank" href="$(canonical_url('/'))">openlibrary.org</a></p>
+      <p class="service">on <a target="_blank" href="$(canonical_url('/'))">openlibrary.org')</a></p>
```

Comment:
This is what this actually looks like
![image](https://github.com/internetarchive/openlibrary/assets/6251786/f5b05749-0420-46ac-85c5-cf90aee5f25f)

So I think we'll want the "Learn More" in one translation, and the "on openlibrary.org" in one translation.

Reply:
Another conflicting comment! +1 Drini's right you had it right the first time